src/components/Composer/index.native.js

```diff
@@ -78,6 +79,9 @@ class Composer extends React.Component {
             return;
         }
 
+        // We want this to be an available method on the ref for parent components
+        this.textInput.setTextAndSelection = this.setTextAndSelection;
```

Comment:
Technically, I don't mind this change. This is clean and does not affect a bigger portion of the app.